### PR TITLE
Update itertools.

### PR DESCRIPTION
**Pull request by emilio**
This allows me to update it in Gecko without duplicating the dependency.

### CODE REVIEW

File: crates/binjs_generate_library/src/lib.rs [hunk 1]
```diff
@@ -347,9 +347,8 @@ impl<'a> Walker<'a> for {name} where Self: 'a {{
             let mut options = vec![];
             let mut lists = vec![];
             let mut primitives = vec![];
-            let names: Vec<_> = source.keys().sorted();
 
-            for name in names.into_iter() {
+            for name in source.keys().sorted() {
```
Comment (Yoric) [inline crates/binjs_generate_library/src/lib.rs]: Oh, they changed the return type of `sorted()`? Fun times.

**Comment by emilio** [comment]
Yeah, a new version would be appreciated! :)

**Comment by Yoric** [comment]
@emilio it doesn't seem to pass tests, can you check?

**Comment by emilio** [comment]
I had missed a `--all` in my cargo test command. Should be good now :)

**Comment by Yoric** [comment]
Thanks :)

**Comment by Yoric** [comment]
New version released.

**Comment by Yoric** [comment]
@emilio I'm updating SpiderMonkey to use the latest version but `./mach rust vendor` still claims that 0.7.x is used somewhere.

**Comment by emilio** [comment]
Yes, it's used in the style system, I can take care to update it and you can review, or vice versa I guess :)

**Comment by emilio** [comment]
As far as I can tell, updating itertools in the style crate should require no code changes at all, so I'm happy to rubber-stamp the version increment.

**Pull request merged**